app/assets/javascripts/discourse/app/components/sidebar/section.hbs

```diff
@@ -1,39 +1,21 @@
 <div class={{concat "sidebar-section-wrapper sidebar-section-" @sectionName}}>
-  <div class="sidebar-section-header">
-    {{#if @collapsable}}
-      <button
-        type="button"
-        class="sidebar-section-header-caret"
-        title="toggle section"
-        {{on "click" this.toggleSectionDisplay}}
-      >
-        {{d-icon this.headerCaretIcon}}
-      </button>
-    {{/if}}
-
-    {{#if @headerRoute}}
-      <LinkTo
-        @route={{@headerRoute}}
-        @query={{@headerQuery}}
-        @models={{if
-          @headerModel
-          (array @headerModel)
-          (if @headerModels @headerModels (array))
-        }}
-        class="sidebar-section-header-link"
-        title={{@headerLinkTitle}}
-      >
-
-        {{@headerLinkText}}
-      </LinkTo>
-    {{else}}
-      <span
-        title={{@headerLinkTitle}}
-        class="sidebar-section-header-text"
-      >
+  <div class="sidebar-section-header-wrapper sidebar-row">
+    <span
+      role="button"
+      title={{i18n "sidebar.toggle_section"}}
+      class="sidebar-section-header"
+      {{on "click" this.toggleSectionDisplay}}
```

Comment:
Do we need to check if `@collapsable` ?

Reply:
Let me defer this one first because I don't think there is an easy way to set a conditional action. Adding the conditional in the action feels wrong so I'm not sure what I can do :thinking:

Reply:
I was thinking that we should replace whole tag. Something like:
```javascript
if(@collapsable) { 
  clickable span with button role and mouse indicating that it is clickable
} else {
  just text span without clickable mouse cursor
}
```

Reply:
Done :) I did this by adding a `<Sidebar::SectionHeader>` component.